### PR TITLE
Mkr1300 build

### PR DESCRIPTION
**Pull request by pelikhan**
Setting up the build for MKR1300. @sabas1080 please review.

### CODE REVIEW

File: pxtarget.json [hunk 1]
```diff
@@ -123,6 +123,44 @@
...
+                "openocdScript": "source [find interface/cmsis-dap.cfg]; set CHIPNAME at91samd21g18; source [find target/at91samdXX.cfg]",
+                "flashChecksumAddr": 8372,
+                "flashCodeAlign": 256,
+                "upgrades": []
+            },            
+            "compileService": {
+                "buildEngine": "codal",
+                "codalTarget": {
+                    "name": "codal-mkr1300",
+                    "url": "https://github.com/ElectronicCats/codal-mkr1300",
```
Comment (sabas1080) [inline pxtarget.json]: @pelikhan I do not know whether to place the original repository https://github.com/lancaster-university/codal-mkr1300 or the ElectronicCats repository, we as electroniccats can maintain our repository

**Comment by sabas1080** [comment]
@pelikhan I am doing testing and do not finish compiling with polling
captura de pantalla 2018-10-19 a la s 12 32 04

File: pxtarget.json (hunk 1 above)
Comment (sabas1080) [reply pxtarget.json]: ok, for me it's okay

Comment (pelikhan) [reply pxtarget.json]: seems to work for me.

**Pull request merged**